### PR TITLE
Added Bridge edge in Graph in python

### PR DESCRIPTION
**Pull request by atriadhiakri2000**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ Y ] I've read the contribution guidelines.
- [ Y ] I've checked the issue list before deciding what to submit.
- [ Y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by atriadhiakri2000** [comment]
@iamrajiv Please Review my code

File: Python/README.md [hunk 1]
```diff
@@ -63,7 +63,6 @@
 - [Trie](ds/Trie.py)
 - [Floyd Warshall Algorithm](graphs/Floyd_Warshall_Algorithm.py)
 - [Knight Walk](graphs/Knight_Walk.py)
-- [Bridge Edge in Graph](graph/Bridge_Edge_in_Graph.py)
```
Comment (iamrajiv) [inline Python/README.md:66]: @atriadhiakri2000 Do not remove the program link.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/graphs/Bridge_Edge_in_Graph.py [hunk 2]
```diff
@@ -109,16 +112,19 @@ def checkbridge(graph, no_of_vertices, u, v):
...
+    print ("The BRIDGE is present.")
 else:
-    print (False)
+    print ("The BRIDGE is not present.")
 
 # SAMPLE
 '''
+INPUT
 
 No of Vertices = 4
```
Comment (atarax665) [inline Python/graphs/Bridge_Edge_in_Graph.py]: Update Sample Input according to the updated code.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Update the deleted line in README.md as ```[Bridge Edge in Graph](graphs/Bridge_Edge_in_Graph.py)```

**Comment by atriadhiakri2000** [comment]
@iamrajiv  Please check
@atarax665  Please check

**Review by atarax665 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**